ui/src/components/modal/BusinessServiceModal.js

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
Do we need a separate modal for businessservice and we cant use the existing modal?